glide.yaml

```diff
@@ -25,7 +25,7 @@ import:
   subpackages:
   - /difflib
 - package: github.com/elastic/gosigar
-  version: cabc0737dc6b82977febbc5c24865259447e1965
+  version: 5f96f2b3d1ee9e656c2951522fa9b64c26951b2e
```

Comment:
I squashed the commit in gosigar, and the hash changed. Can you update this PR, please?